admin/command_runner_test.go

```diff
@@ -25,19 +26,22 @@ import (
 	"github.com/stretchr/testify/suite"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
+	grpcinsecure "google.golang.org/grpc/credentials/insecure"
 	"google.golang.org/grpc/status"
 	"google.golang.org/protobuf/types/known/structpb"
 
 	pb "github.com/onflow/flow-go/admin/admin"
 	"github.com/onflow/flow-go/module/irrecoverable"
+	"github.com/onflow/flow-go/utils/unittest"
 )
 
 type CommandRunnerSuite struct {
 	suite.Suite
 
-	runner       *CommandRunner
-	bootstrapper *CommandRunnerBootstrapper
-	httpAddress  string
+	runner          *CommandRunner
+	bootstrapper    *CommandRunnerBootstrapper
+	httpAddress     string
+	grpcAddressSock string
```

Comment:
these tests were failing frequently during local testing. Seemed to be an issue with socket file collisions, possibly because the file wasn't being cleaned up.